mico-core/src/main/java/io/github/ust/mico/core/dto/request/MicoApplicationRequestDTO.java

```diff
@@ -60,12 +61,12 @@
             @ExtensionProperty(name = "title", value = "Short Name"),
             @ExtensionProperty(name = "pattern", value = Patterns.KUBERNETES_NAMING_REGEX),
             @ExtensionProperty(name = "minLength", value = "3"),
-            @ExtensionProperty(name = "maxLength", value = "253"),
+            @ExtensionProperty(name = "maxLength", value = KubernetesNameNormalizer.MAX_LABLE_SIZE+""),
```

Comment:
`+""`?

Reply:
The value hast to be a string while the constant is an int.